azurerm/internal/services/policy/tests/policy_assignment_resource_test.go

```diff
@@ -241,6 +241,11 @@ resource "azurerm_policy_assignment" "test" {
   name                 = "acctestpa-%[1]d"
   scope                = azurerm_resource_group.test.id
   policy_definition_id = azurerm_policy_definition.test.id
+  metadata = <<METADATA
```

Comment:
And actually if you add an empty line before this line, the tf config should be considered as formatted

Reply:
done. added an empty line before this line.